content/hardware/04.pro/boards/portenta-h7/datasheets/datasheet.md

```diff
@@ -484,16 +484,18 @@ Laboratory equipment, Computer vision
 ### Recommended Operating Conditions
 
 | Symbol | Description                                      | Min             | Max            |
-| ------ | ------------------------------------------------ | --------------- | -------------- |
+|--------|--------------------------------------------------|-----------------|----------------|
 |        | Conservative thermal limits for the whole board: | -40 °C (-40 °F) | 85 °C (185 °F) |
 
 ### Power Consumption
 
-| Symbol  | Description                              | Min  | Typ  | Max  | Unit |
-| ------- | ---------------------------------------- | ---- | ---- | ---- | ---- |
-| VINMax  | Maximum input voltage from VIN pad       | -0.3 | -    | xx   | V    |
-| VUSBMax | Maximum input voltage from USB connector | -0.3 | -    | xx   | V    |
-| PMax    | Maximum Power Consumption                | -xx  | -    | xx   | mW   |
+| Description                                                 | Min | Typ  | Max | Unit |
+|-------------------------------------------------------------|-----|------|-----|------|
+| Power consumption in deep sleep mode with 3.7V power supply |     | 0.67 |     | mA   |
+| Power consumption in busy loop with 5V power supply         |     | 2.6  |     | mAh  |
+| Maximum power consumption with 5V power supply              |     | 230  |     | mA   |
+
+**Note:** All the measurements have been performed with Portenta H7 full version (ABX00042). The busy loop taken as reference consists of 595 seconds of deep sleep time and 5 seconds of power on and radio on. 
```

Comment:
Is the 2.6mA mentioned an average over the 5 seconds or across the whole 595+5 seconds?

Reply:
I added a line in the note, let me know if now is more clear, thanks!